tests/plugins/test_change_from_in_df.py

```diff
@@ -23,18 +23,23 @@
 from textwrap import dedent
 
 
-def mock_workflow():
+def mock_workflow(image_name=None, dockerfile_path=None, parent_name=None, parent_images=None):
```

Comment:
This change is reversing the [test data builder pattern](https://testing.googleblog.com/2018/02/testing-on-toilet-cleanly-create-test.html) that we wanted to follow.

Reply:
Fixed.